### PR TITLE
[UXD][AAP-25027] AWX - updates to Credentials & Cred Types section breadcrumbs & headers

### PR DESCRIPTION
**Pull request by tiyiprh**
https://issues.redhat.com/browse/AAP-25027
Updates to Credentials & Cred Types section breadcrumbs & headers

### CODE REVIEW

File: cypress/e2e/awx/administration/credentialTypes.cy.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ describe('Credential Types', () => {
 
     it('can create a new credential type with no configs', () => {
       cy.createAndDeleteCustomAWXCredentialTypeUI(customCredentialTypeName);
-      cy.verifyPageTitle('Credential Types');
+      cy.verifyPageTitle('Create credential type');
```
Comment (acruzgon) [inline cypress/e2e/awx/administration/credentialTypes.cy.ts:140]:
```suggestion
      cy.verifyPageTitle('Credential Types');
```

File: cypress/e2e/awx/administration/credentialTypes.cy.ts [hunk 2]
```diff
@@ -147,7 +147,7 @@ describe('Credential Types', () => {
         injectorCredType,
         'json'
       );
-      cy.verifyPageTitle('Credential Types');
+      cy.verifyPageTitle('Create credential type');
```
Comment (acruzgon) [inline cypress/e2e/awx/administration/credentialTypes.cy.ts:150]:
```suggestion
      cy.verifyPageTitle('Credential Types');
```

File: cypress/e2e/awx/administration/credentialTypes.cy.ts [hunk 3]
```diff
@@ -156,7 +156,7 @@ describe('Credential Types', () => {
         inputCredType,
         injectorCredType
       );
-      cy.verifyPageTitle('Credential Types');
+      cy.verifyPageTitle('Create credential type');
```
Comment (acruzgon) [inline cypress/e2e/awx/administration/credentialTypes.cy.ts:159]:
```suggestion
      cy.verifyPageTitle('Credential Types');
```

**Review by acruzgon — CHANGES_REQUESTED** [review]
1. Go to `/cypress/support/awx-commands.ts`
2. In line 288 replace with `cy.verifyPageTitle('Create credential type');`

**Comment by tiyiprh** [comment]
@daphnemaeve There were 4 failing previously yesterday and then I did a rebase and now there are still 2 failing.

File: cypress/e2e/awx/access/credential-types/credentialTypes.cy.ts [hunk 4]
```diff
@@ -137,7 +137,7 @@ describe('Credential Types', () => {
 
     it('can create a new credential type with no configs', () => {
       cy.createAndDeleteCustomAWXCredentialTypeUI(customCredentialTypeName);
-      cy.verifyPageTitle('Credential Types');
+      cy.verifyPageTitle('Create credential type');
```
Comment (acruzgon) [inline cypress/e2e/awx/access/credential-types/credentialTypes.cy.ts:140]:
```suggestion
      cy.verifyPageTitle('Create Types');
```

**Review by acruzgon — CHANGES_REQUESTED** [review]

File: cypress/e2e/awx/access/credential-types/credentialTypes.cy.ts [hunk 5]
```diff
@@ -137,7 +137,7 @@ describe('Credential Types', () => {
 
     it('can create a new credential type with no configs', () => {
       cy.createAndDeleteCustomAWXCredentialTypeUI(customCredentialTypeName);
-      cy.verifyPageTitle('Credential Types');
+      cy.verifyPageTitle('Create Types');
```
Comment (acruzgon) [inline cypress/e2e/awx/access/credential-types/credentialTypes.cy.ts:140]: sorry i had a typo! this is `Credential Types`

**Review by acruzgon — CHANGES_REQUESTED** [review]

File: cypress/e2e/awx/access/credential-types/credentialTypes.cy.ts (hunk 5 above)
Comment (tiyiprh) [reply cypress/e2e/awx/access/credential-types/credentialTypes.cy.ts]: no worries - just updated!

**Review by acruzgon — APPROVED** [review]
lgtm!

**Review by nixocio — APPROVED** [review]

**Pull request merged**